magni_bringup/param/core_launch.em

```diff
@@ -0,0 +1,67 @@
+<launch>
+
+    <arg name="sonars_installed" default="@(sonars_installed)"/>
+    <arg name="oled_display" default="@(oled_display)"/>
+    <arg name="controller_board_version" default="@(controller_board_version)"/>
+    
+    <include file="$(find magni_description)/launch/description.launch">
+        <arg name="camera_extrinsics_file" value="@(camera_extrinsics_file)"/>
+        <arg name="lidar_extrinsics_file" value="@(lidar_extrinsics_file)"/>
+        <arg name="sonars_installed" value="$(arg sonars_installed)"/>
+    </include>
+
+    <group if="$(arg sonars_installed)">
```

Comment:
This is ok for now, but we might want to move some of the the logic into empy for more flexibility than the rigid roslaunch logic.

Reply:
yeah, true, otherwise, whats the point of generating it with python :D